blocks/library/spacer/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress
+ */
+import { __ } from '@wordpress/i18n';
+import ResizableBox from 're-resizable';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import InspectorControls from '../../inspector-controls';
+
+export const name = 'core/spacer';
+
+export const settings = {
+	title: __( 'Spacer' ),
+
+	description: __( 'Add an element with empty space and custom height.' ),
+
+	icon: 'image-flip-vertical',
+
+	category: 'layout',
+
+	attributes: {
+		height: {
+			type: 'number',
+			default: 100,
+		},
+	},
+
+	edit( { attributes, setAttributes, isSelected, toggleSelection } ) {
+		const { height } = attributes;
+
+		return [
+			<ResizableBox
+				size={ {
+					height: height,
```

Comment:
Minor: Can take advantage of object property shorthand notation:

```js
{
	height,
}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015